packages/ag-charts-community/src/chart/noopAgChartInstance.ts

```diff
@@ -0,0 +1,38 @@
+import type { AgChartOptions, AgChartState } from 'ag-charts-types';
+
+const errorMessage = 'AG Charts - Method not implemented.';
+export class NoopAgChartInstance {
```

Comment:
I'm actually wondering if we should remove this, and just make the `Chart.publicApi` property optional?

The call in the `ContextMenu` type then becomes:
```
await this.ctx.chartService.publicApi?.download({ fileName });
```